apps/teams-test-app/src/components/StoreApis.tsx

```diff
@@ -0,0 +1,49 @@
+import { store } from '@microsoft/teams-js';
+import { ReactElement } from 'react';
+import React from 'react';
+
+import { ApiWithoutInput, ApiWithTextInput } from './utils';
+import { ModuleWrapper } from './utils/ModuleWrapper';
+
+const StoreAPIs = (): ReactElement => {
+  const CheckStoreCapability = (): ReactElement =>
+    ApiWithoutInput({
+      name: 'checkCapabilityStore',
+      title: 'Check Capability Store',
+      onClick: async () => {
+        if (store.isSupported()) {
+          return 'Store module is supported';
+        } else {
+          return 'Store module is not supported';
+        }
+      },
+    });
+
+  const OpenStore = (): ReactElement =>
```

Comment:
Before checking in, you will need to write a json file defining end-to-end tests for this capability. You can see other examples of these in our repo (for example, hostEntity.tab.json. If you have any questions about how write these, I will also forward you some information about how to learn more.

Reply:
added :)